SIPS/sip-29.md

```diff
@@ -117,15 +117,13 @@ type Conversion = {
 
 type OnAssetsConversionArguments = {
   conversions: Conversion[];
-  includeMarketData?: boolean;
```

Comment:
Do we no longer need to pass this arg from the controller? is the snap going to automatically return market data for assets now?

Reply:
No we moved the market data to a new handler, we will have to call this new handler in the controller

Reply:
https://github.com/MetaMask/SIPs/blob/78ce26983b202dfb3828f4948972b62c07a66587/SIPS/sip-29.md?#L149-L187